torch/csrc/jit/codegen/cuda/kernel_ir.cpp

```diff
@@ -107,6 +107,20 @@ CpAsyncCommit::CpAsyncCommit(IrBuilderPasskey passkey)
       "IR type only valid for Kernel container.");
 }
 
+AddressCompute::AddressCompute(
+    IrBuilderPasskey passkey,
+    AddressCompute::AddressComputeOpType op_type,
+    Val* address_tensor,
+    Val* data_tensor)
+    : Expr(passkey, ExprType::AddressCompute),
+      op_type_(op_type),
+      data_tensor_(data_tensor),
+      address_tensor_(address_tensor) {
+  TORCH_INTERNAL_ASSERT(
+      passkey.ir_container_->isA<kir::Kernel>(),
+      "IR type only valid for Kernel container.");
+}
```

Comment:
Not sure but do we want to set inputs and outputs of this expression? Conceptually, the tensor whose address is computed can be considered an input, and the tensor storing the address seems like an output.